documentation/plain_rules.md

```diff
@@ -193,6 +194,7 @@ The `parameters` map includes the following fields:
 -   actionType: optional, type of CB action: APPEND or UPDATE. By default is APPEND.
 -   trust: optional, trust token for getting an access token from Auth Server which can be used to get to a Context
     Broker behind a PEP.
+-   filter: optional, a NGSI filter. If provided then updateAction is done over result of query.
```

Comment:
In addition, I'd suggest to modify the version field this way:

```
-   version: optional, The NGSI version for the update action. Set this attribute to `2` or `"2"` if you want to use
    NGSv2 format. `1` by default. However, if `filter` is used, this field is ignored and NGSIv2 is used.
```

Reply:
fixed in d93078c1a87dfd9e54e00f3021a06c9893620b11